client/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
     "fetch": "^1.1.0",
+    "jwt-decode": "^2.2.0",
```

Comment:
Why was this committed? The dependencies were already set up in the previous PR.

Reply:
This was autmatically changed when I was adding more dependencies that I needed, but the change is probably irrelevant

Reply:
I understand, since the package.json automatically updates whenever a dependency is added. However, it isn't required to commit, as it can potentially change the current running version a different one that can cause some problems.